css-brochure-form/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <h1>LEARN MORE</h1>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column-full center">
+        <h2>COMPANY EMAIL</h2>
+        <input type="text" class="form" placeholder="sally@company.co">
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column-full center">
+        <h2>NAME</h2>
+        <input type="text" class="form" placeholder="Sally Smith">
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column-full center">
+        <h2>WHERE ARE YOU HIRING?</h2>
+        <input type="text" class="form" placeholder="City, Country">
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column-full center">
+        <h2>HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</h2>
+        <select name="engineers" class="form">
+          <option value="option 1">1-2 Engineers</option>
+          <option value="option 2">2-3 Engineers</option>
+          <option value="option 3">3-4 Engineers</option>
+          <option value="option 4">5+ Engineers</option>
+        </select>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="button-column">
```

Comment:
Instead of defining another `button-column`, we can do `class="column-full end"` => See another comment in CSS file about `end`